pkg/loader/loader.go

```diff
@@ -30,6 +30,7 @@ var embeddedPluginNames = []string{
 }
 
 var dir = "./plugins"
+var typesDir = "./plugins/types"
```

Comment:
This is a tiny detail, but I wonder should it be "generated" or ".generated" or ".cache"? Something to say that this is a dir of transient things.